entries/n/nelnet.studentaid.gov.json

```
@@ -0,0 +1,12 @@
+{
+  "Nelnet": {
+    "domain": "nelnet.studentaid.gov",
+    "contact": {},
```

```suggestion
    "contact": {
      "form": "https://nelnet.studentaid.gov/content/contactus"
    },
```

They have a live chat option.

---

Good find!